pkg/apis/app/v2/kieapp_types.go

```diff
@@ -977,6 +979,20 @@ type KafkaExtObject struct {
 	MaxBlockMs *int32 `json:"maxBlockMs,omitempty"`
 }
 
+// KafkaJBPMEventEmittersObject kafka configuration to be used by the KieApp for JBPM Emitter
+type KafkaJBPMEventEmittersObject struct {
+	// Jbpm emitter, a comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster for Jbpm Emitter
+	BootstrapServers string `json:"bootstrapServers,omitempty"`
+	// Jbpm Emitter, identifier to pass to the server when making requests
+	ClientID string `json:"clientID,omitempty"`
+	// Jbpm Emitter, the number of acknowledgments the producer requires the leader to have received before considering a request complete.
+	Acks *int `json:"acks,omitempty"`
+	// Jbpm emitter, number of milliseconds that indicates how long publish method will bloc for jbpm emitter, default value is 2000
+	MaxBlockMs *int32 `json:"maxBlockMs,omitempty"`
+	// Date and time format to be sent to Kafka - default format is yyyy-MM-dd'T'HH:mm:ss.SSSZ
+	DateFormat string `json:"dateFormat,omitempty"`
```

Comment:
isn't missing a this property?
https://github.com/jboss-container-images/jboss-kie-modules/pull/456/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cR42

Reply:
No, we add in the common.yaml if the KafkaJbpmEventEmitters is present https://github.com/kiegroup/kie-cloud-operator/pull/654/files#diff-66b9e589dd9cc4038e91fe574a84c9d024cc6e65956366412f929a815e43cebaR946

Reply:
ok